libpod/container_internal_common.go

```diff
@@ -2827,8 +2827,6 @@ func (c *Container) fixVolumePermissions(v *ContainerNamedVolume) error {
 	// Volumes owned by a volume driver are not chowned - we don't want to
 	// mess with a mount not managed by us.
 	if vol.state.NeedsChown && (!vol.UsesVolumeDriver() && vol.config.Driver != "image") {
-		vol.state.NeedsChown = false
```

Comment:
this field is exposed with volume inspect and has public documentation so that must be changed

I think always doing the chown should not be a breaking change so it is fine but if you have more plans to change the volume behaviour then we should be really careful to not break existing podman users

Reply:
Make sure it is a safechown, don't change uid=0 to uid=0.